pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -857,7 +833,7 @@ private PantheonCommand controller() {
     return this;
   }
 
-  PantheonController<?> buildController() {
+  public PantheonController<?> buildController() {
```

Comment:
You've got a lot of visibility changes here - looks like at least a few of these are unnecessary.  Can you double-check the visibility changes?

Reply:
I have double checked. They are all necessary because sub commands are not in  the same package as the parent command. I have only made public methods that are actually used by sub commands.